script/repository-list.txt

```diff
@@ -1,24 +1,22 @@
 droolsjbpm-build-bootstrap
 kie-soup
-droolsjbpm-knowledge
 drools
-optaplanner
 jbpm
-kie-jpmml-integration
-droolsjbpm-integration
-openshift-drools-hacep
 lienzo-core
 lienzo-tests
 appformer
 kie-uberfire-extensions
+kie-jpmml-integration
+droolsjbpm-integration
+openshift-drools-hacep
 kie-wb-playground
 kie-wb-common
 drools-wb
-optaplanner-wb
 jbpm-designer
 jbpm-work-items
 jbpm-wb
-kie-docs
-optaweb-employee-rostering
-optaweb-vehicle-routing
 kie-wb-distributions
+optaplanner-wb
+droolsjbpm-tools
+droolsjbpm-knowledge
+kie-docs
```

Comment:
this order seems like not correct to me.

Reply:
in this case the dependencies from https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/713c3c327c197f1ec9cdf692a337bb70f6f86ff4/.ci/project-dependencies.yaml#L140 are wrong. Are they needed by `kie-wb-distributions`?

Reply:
I thought that they are, if not, could we have a test build to confirm it?

Reply:
@mareknovotny this would be the repository-list.txt order after applying https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1540
`droolsjbpm-knowledge` has been moved up

```
droolsjbpm-build-bootstrap
kie-soup
droolsjbpm-knowledge
drools
optaplanner
lienzo-core
lienzo-tests
appformer
kie-uberfire-extensions
jbpm
kie-jpmml-integration
droolsjbpm-integration
kie-wb-playground
kie-wb-common
drools-wb
jbpm-designer
jbpm-work-items
jbpm-wb
kie-wb-distributions
optaplanner-wb
openshift-drools-hacep
optaweb-employee-rostering
optaweb-vehicle-routing
droolsjbpm-tools
kie-docs
```

Reply:
this repo list looks good

Reply:
but my concern was about kie-wb-distribution not on the last position/line

Reply:
changes test by https://github.com/kiegroup/kie-wb-distributions/pull/1073

Reply:
https://github.com/kiegroup/kie-wb-distributions/pull/1073/checks?check_run_id=1472535219 worked